wremnants/plot_tools.py

```diff
@@ -284,6 +284,8 @@ def makeStackPlotWithRatio(
             data_idx = unstacked.index("Data") 
             linestyles[data_idx] = "None"
         linestyles = np.array(linestyles, dtype=object)
+        print("Number of linestyles", len(linestyles))
+        print("Length of unstacked", len(unstacked))
```

Comment:
If another commit is added for a different reasons the print statements could be replaced by logging

Reply:
I will add more commits. I am playing around here, if it's causing issues just kill the CI. Probably not worth making careful comments